### PR TITLE
Create 37i9dQZF1DX5inthBIooNX

### PR DESCRIPTION
**Pull request by user41377**

### CODE REVIEW

**Comment by mackorone** [comment]
> FAILED TO EXTRACT AUTHOR

cc @maciejpedzich

**Comment by maciejpedzich** [comment]
> > FAILED TO EXTRACT AUTHOR
> 
> cc @maciejpedzich

On it

**Comment by maciejpedzich** [comment]
**TL;DR** - It looks like I've fixed the bug

@mackorone my initial implementation incorrectly assumed the playlist's page title ended with `- playlist by $USER$ | Spotify` for user-made playlists or `Spotify Playlist` for autogenerated ones, but that substring appears to get appended by client-side JavaScript code, which doesn't get loaded by the bot, since it only looks at raw HTML.

The HTML in question contains a tag with a URL of author's Spotify page, from which I can finally grab author's name by looking at its title tag, unless the URL ends with `/user/spotify`, meaning I can safely assume that `Spotify` is the author and skip sending another request entirely.
